autonomy/deploy/image.py

```diff
@@ -35,7 +35,6 @@
 )
 from autonomy.data import DATA_DIR
 from autonomy.deploy.constants import DOCKERFILES
-from autonomy.deploy.generators.docker_compose.base import get_docker_client
```

Comment:
amazing!

This will make us compatible with 3.13 and also removes the depndency

Im very happy

Reply:
I am very very very happy about this.

Please see pr;

https://github.com/8ball030/auto_dev/pull/538

AFAICT you guys relied on requests and the forked repo you have is not oss

Reply:
It really would be sick if we could get ready for 3.12 support.

AFAICT this pr 
https://github.com/valory-xyz/open-autonomy/pull/2316

here should basiclaly get us there with the 2 deps;
tomte[flake8]
and 
open-aea-cosmpy being updated.